components/prism-sqf.js

```diff
@@ -0,0 +1,33 @@
+Prism.languages.sqf = Prism.languages.extend('clike', {
+	'string': {
+		pattern: /"(?:(?:"")?[^"])*"(?!")|'(?:[^'])*'/,
+		greedy: true
+	},
+	'keyword': /\b(?:call|case|catch|default|do|echo|else|endsWith|for|forEach|from|goto|if|nil|private|step|switch|then|to|try|while)\b/,
```

Comment:
```suggestion
	'keyword': /\b(?:call|case|catch|default|do|echo|else|exitWith|for|forEach|from|goto|if|nil|private|step|switch|then|to|try|while)\b/,
```

Should also be removed from the "function" list below as I think this should be a keyword (like JonBons/Sublime-SQF-Language).

Reply:
Yes, definitely. Seems like I simply mistyped `exitWith`.